browser/components/MarkdownPreview.js

```diff
@@ -26,7 +26,7 @@ const dialog = remote.dialog
 const markdownStyle = require('!!css!stylus?sourceMap!./markdown.styl')[0][1]
 const appPath = 'file://' + (process.env.NODE_ENV === 'production'
   ? app.getAppPath()
-  : path.resolve())
+  : path.resolve().replace(/\\/g, '/'))
```

Comment:
What is this doing? May it cause problems on windows maschines, where there is another separator?

Reply:
This is to solve a problem on windows.  This replaces all back-slashes (\\) with forward-slashes (/). Without this, in dev mode, on windows, it's broken and some of the fonts do not get loaded. On windows, you end up with a mixture of slashes since the appPath starts with `file://`, but `path.resolve()` obviously returns windows style paths containing back slashes.

Reply:
Okay, Thanks for the information :)